sessions/react-state-3/journal-app-filter/src/components/Entry/index.js

```diff
@@ -0,0 +1,30 @@
+import FavoriteButton from "../FavoriteButton";
+import "./Entry.css";
+
+export default function Entry({
+  motto,
+  notes,
+  date,
+  id,
+  isFavorite,
+  onToggleFavorite,
+}) {
+  return (
+    <article className="entry">
+      <time className="entry__date">{date}</time>
+      <div className="entry__content">
+        <div className="entry__motto-container">
+          <h2 className="entry__motto">
+            <q>{motto}</q>
+          </h2>
+          <FavoriteButton
+            id={id}
+            isFavorite={isFavorite}
+            onToggleFavorite={onToggleFavorite}
```

Comment:
```suggestion
            onToggleFavorite={onToggleFavorite}
```
Same here, perhaps. Shouldn't this be passed as `handleToggleFavorite`? Maybe I've misunderstood something.

Reply:
If you're fully pedantic yes - but since we're just passing it down we should be fine without an intermediary function just to rename it… the overhead would be worse than just passing it down